src/VASL/build/module/map/boardPicker/ASLBoard.java

```diff
@@ -125,26 +125,27 @@ public String getLocalizedName() {
      */
     public void initializeFromArchive(File archiveFile) {
 
-        try {
+        try{
             VASLBoardArchive = new BoardArchive(archiveFile.getName(), archiveFile.getParent(), ASLMap.getSharedBoardMetadata());
-
         } catch (IOException e) {
             ErrorDialog.dataWarning(new BadDataReport("Unable to open board file", archiveFile.getName(), e));
+            return;
         }
-        try {
-            boardFile = archiveFile;
-            setCommonName(VASLBoardArchive.getBoardName());
-            imageFile = VASLBoardArchive.getBoardImageFileName();
-            version = VASLBoardArchive.getVersion();
 
+        boardFile = archiveFile;
+        setCommonName(VASLBoardArchive.getBoardName());
+        imageFile = VASLBoardArchive.getBoardImageFileName();
+        version = VASLBoardArchive.getVersion();
+        if (version == null){version="0.0";}
+        try {
             ((Translatable) getGrid()).setAttribute(HexGrid.X0, (int) VASLBoardArchive.getA1CenterX());
             ((Translatable) getGrid()).setAttribute(HexGrid.Y0, (int) VASLBoardArchive.getA1CenterY());
             ((Translatable) getGrid()).setAttribute(HexGrid.DX, VASLBoardArchive.getHexWidth());
             ((Translatable) getGrid()).setAttribute(HexGrid.DY, VASLBoardArchive.getHexHeight());
             ((Translatable) getGrid()).setAttribute(HexGrid.SNAP_SCALE, VASLBoardArchive.getSnapScale());
             ((Translatable) getGrid()).setAttribute(BoardArchive.ALT_HEX_GRID_KEY, Boolean.toString(VASLBoardArchive.isAltHexGrain()));
         } catch (Exception e) {
```

Comment:
What `Exception` are you expecting here?

Reply:
I did not create that try catch. I simply moved the lines above it out of the try catch and handled the one error I know can arise even if VASLBoardArchive is not null  - which is a missing version. 

I suspect that there is no need for the try catch but I left it there because it was there.

Reply:
It's guaranteed to be masking a problem if it ever runs. I suggest removing the try-catch.

Reply:
In other words: You _want_ a crash here if there's an exception, so you can find out what the bug is.

Reply:
Done. I will push the change.